sub-packages/bionemo-llm/src/bionemo/llm/train.py

```diff
@@ -71,6 +71,7 @@ def nemo_logger_factory(experiment_config: ExperimentConfig, wandb_config: Optio
         save_top_k=experiment_config.save_top_k,
         every_n_train_steps=experiment_config.save_every_n_steps,
         always_save_context=True,
+        filename='{epoch}-{val_loss:.2f}-{step}-{consumed_samples}'
```

Comment:
Same comment here about step/consumed samples being included to ensure unique names. Just make sure people think about this when they see this line incase they are considering changing it.

Reply:
done